coldfront/config/email.py

```diff
@@ -22,3 +22,4 @@
 EMAIL_ALLOCATION_EXPIRING_NOTIFICATION_DAYS = ENV.list('EMAIL_ALLOCATION_EXPIRING_NOTIFICATION_DAYS', cast=int, default=[7, 14, 30])
 EMAIL_SIGNATURE = ENV.str('EMAIL_SIGNATURE', default='', multiline=True)
 EMAIL_ADMINS_ON_ALLOCATION_EXPIRE = ENV.bool('EMAIL_ADMINS_ON_ALLOCATION_EXPIRE', default=False)
+EMAIL_EULA_REMINDERS = ENV.bool('EMAIL_EULA_REMINDERS', default=True)
```

Comment:
Same here, rename to `EMAIL_ALLOCATION_EULA_REMINDERS` and let's default to False

Reply:
Done